dir.targets

```diff
@@ -28,5 +28,15 @@
     <TargetFrameworkProfile></TargetFrameworkProfile>
     <!-- We set this property to avoid MSBuild errors regarding not setting TargetFrameworkProfile (see above line) -->
     <PortableNuGetMode>true</PortableNuGetMode>
+
```

Comment:
This needs to be reconciled with @Priya91 recent BuildTools PR https://github.com/dotnet/buildtools/pull/564